exercises/sieve/instructions.md

```diff
@@ -5,24 +5,36 @@ Your task is to create a program that implements the Sieve of Eratosthenes algor
 A prime number is a number that is only divisible by 1 and itself.
 For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
 
-The Sieve of Eratosthenes is an ancient algorithm that works by taking a list of numbers and crossing out all the numbers that aren't prime.
-
-A number that is **not** prime is called a "composite number".
-
 To use the Sieve of Eratosthenes, you first create a list of all the numbers between 2 and your given number.
```

Comment:
«your given number»: no given number was mentioned previously. No biggie, but could be fixed using with e.g. (above)

```diff
-Your task is to create a program that implements the Sieve of Eratosthenes algorithm to find prime numbers.
+Your task is to create a program that implements the Sieve of Eratosthenes algorithm to find all prime numbers below a given number.
```